### PR TITLE
Bing Maps Labels Only

### PR DESCRIPTION
**Pull request by mramato**
# Description

Added a new Sandcastle example, Bing Maps Labels Only, which demonstrates using the Cesium ion imagery asset of the same name. Local link: http://localhost:8080/Apps/Sandcastle/index.html?src=Bing%20Maps%20Labels%20Only.html&label=New%20in%201.112

## Issue number and link

No number.

## Testing plan

Browse to http://localhost:8080/Apps/Sandcastle/index.html?src=Bing%20Maps%20Labels%20Only.html&label=New%20in%201.112, proofread it, and ensure it works as the comments specify.

# Author checklist

- [x] I have submitted a Contributor License Agreement
- [x] I have added my name to `CONTRIBUTORS.md`
- [x] I have updated `CHANGES.md` with a short summary of my change
- [ ] N/A I have added or updated unit tests to ensure consistent code coverage
- [ ] N/A I have update the inline documentation, and included code examples where relevant
- [X] I have performed a self-review of my code

### CODE REVIEW

File: Apps/Sandcastle/gallery/Bing Maps Labels Only.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    <div id="toolbar"></div>
+
+    <script id="cesium_sandcastle_script">
+      window.startup = async function (Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+        // This example showcases the ability to overlay labels
+        // on top of unlabeled imagery.
+        //
+        // On th left side is Bing Maps Aerial With Labels + unlabeled high
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Bing Maps Labels Only.html:62]:
```suggestion
        // On the left side is Bing Maps Aerial With Labels + unlabeled high
```

File: Apps/Sandcastle/gallery/Bing Maps Labels Only.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use the split property to only show layers on one side of a slider."
+    />
+    <meta
+      name="cesium-sandcastle-labels"
+      content="Beginner, Tutorials, Showcases"
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Bing Maps Labels Only.html:16]:
```suggestion
      content="Beginner, Showcases"
```

"Tutorials" was intended to be used with website tutorials. Do we have a website tutorial planned for this?

File: Apps/Sandcastle/gallery/Bing Maps Labels Only.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
+    />
+    <meta
+      name="description"
+      content="Use the split property to only show layers on one side of a slider."
```
Comment (ggetz) [inline Apps/Sandcastle/gallery/Bing Maps Labels Only.html:12]: Should the description mention labels?

File: CHANGES.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
 
 ### 1.113 - 2024-01-02
 
+- Added a new Sandcastle example, Bing Maps Labels Only, which demonstrates using the Cesium ion imagery asset of the same name.
```
Comment (ggetz) [inline CHANGES.md:5]: Typically, we do not document Sandcastle changes in `CHANGES.md`. We do mention them in release posts and will do so there.

**Review by ggetz — COMMENTED** [review]
Thanks @mramato!  A few comments:

**Comment by mramato** [comment]
Thanks, @ggetz Addressed all comments.

File: Apps/Sandcastle/gallery/Bing Maps Labels Only.html (hunk 3 above)
Comment (mramato) [reply Apps/Sandcastle/gallery/Bing Maps Labels Only.html:12]: Whoops, missed this one. Copy past error.

Comment (mramato) [reply Apps/Sandcastle/gallery/Bing Maps Labels Only.html]: Okay, now it's ready.

**Comment by ggetz** [comment]
Thanks @mramato!

**Pull request merged**